var/spack/repos/builtin/packages/re2c/package.py

```diff
@@ -22,6 +26,15 @@ class Re2c(AutotoolsPackage):
     version("1.3", sha256="f37f25ff760e90088e7d03d1232002c2c2672646d5844fdf8e0d51a5cd75a503")
     version("1.2.1", sha256="1a4cd706b5b966aeffd78e3cf8b24239470ded30551e813610f9cd1a4e01b817")
 
+    phases = ["configure", "build", "install"]
+
+    @property
+    def make(self):
```

Comment:
Can this be called `make_tool` vs. `make`? I think ideally the name would suggest that it isn't always "make" that is being used.

Reply:
I'll be pulling this in favor of the re2c port PR.